### PR TITLE
feat: add unlocode extension

### PR DESCRIPTION
**Pull request by raimundo-henriques**

### CODE REVIEW

File: catalog/data-model-extensions/@sine/unlocode/0.1.0/extension.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "UN/LOCODE Data Model Extension",
+  "files": ["schema.json"],
+  "author": {
+    "name": "SINE Foundation",
+    "email": "pact-catalog@sine.dev",
+    "url": "https://sine.foundation"
+  },
+  "license": "MIT",
+  "catalog_info": {
+    "summary": "The Pathfinder Framework Data Model allows for the interoperable exchange of product footprint data. This includes the product’s carbon footprint, represented in the Data Model as a CarbonFootprint. Each CarbonFootprint is scoped by Geography, specified (at maximum resolution) as a ISO 3166-2 country subdivision. In some cases, the current resolution is unsuficient and a more precise, hopefully plant-level, geographic specification would be recommended. The UN/LOCODE Data Model Extension aims at providing precisely the required level of granularity, using an international coding scheme that is public and can be integrated into solutions.",
```
Comment (fkettelhoit) [inline catalog/data-model-extensions/@sine/unlocode/0.1.0/extension.json:16]: "unsuficient" -> "insufficient"

File: catalog/data-model-extensions/@sine/unlocode/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+```
```
Comment (fkettelhoit) [inline catalog/data-model-extensions/@sine/unlocode/0.1.0/README.md:1]: Wrapping this in a code block means that the line will not be wrapped automatically. Is that really what we want here?

**Review by fkettelhoit — CHANGES_REQUESTED** [review]

File: catalog/data-model-extensions/@sine/unlocode/0.1.0/extension.json (hunk 1 above)
Comment (raimundo-henriques) [reply catalog/data-model-extensions/@sine/unlocode/0.1.0/extension.json:16]: Good catch! This must be fixed also in the documentation

File: catalog/data-model-extensions/@sine/unlocode/0.1.0/README.md (hunk 2 above)
Comment (raimundo-henriques) [reply catalog/data-model-extensions/@sine/unlocode/0.1.0/README.md:1]: That's how it's presented in the original readme, I'm ok with any option tbh

**Review by prasadt1 — APPROVED** [review]

**Pull request merged**